static/app/views/performance/traceDetails/transactionBar.tsx

```diff
@@ -689,3 +691,9 @@ const StyledRow = styled(Row)`
 const ErrorLink = styled(Link)`
   color: ${p => p.theme.error};
 `;
+
+const StyledRowRectangle = styled(RowRectangle)`
+  ${BadgeBorder} {
+    top: -4px;
+  }
+`;
```

Comment:
Negative margins plus nested child selector? That feels a bit weird, does something else need to lose padding instead?